_data/es.yml

```diff
@@ -0,0 +1,344 @@
+  runningRedHat: Parece que está usando Red Hat/Fedora
+  currentVersion: Versión actual
+  hFeatures: Características
+  hIdentityVerifications: Sin verificación de identidad
+  pIdentityVerifications: Accesible instantáneamente. No es necesario registrarse o una aprobación desde una autoridad central.
+  hDecentralized: Descentralizado
+  pDecentralized: Los intercambios se llevan a cabo en una red global P2P de usuarios ejecutando Bisq en sus propias máquinas.
+  hSafe: Seguro
+  pSafe: "Bisq nunca retiene sus fondos. Los depósitos de seguridad y los monederos de intercambio multifirma permiten intercambios seguros y exitosos."
+  hPrivate: privado
+  pPrivate: Nadie excepto los pares de intercambio intercambian datos de identificación personal. Todos los datos se guardan de manera local.
+  hSecure: Seguro y siempre en funcionamiento
+  pSecure: Comunicación cifrada de extremo a extremo enrutada vía Tor y sin caídas.
+  hOpen: Abierto
+  pOpen: "El código es <a href=\"https://github.com/bisq-network/bisq\" target=\"_blank\" rel=\"noopener\">abierto</a>, y la estrategia del proyecto se discute abiertamente en <a href=\"https://bisq.network/slack-invite\" target=\"_blank\" rel=\"noopener\">Slack</a> y <a href=\"https://github.com/bisq-network/bisq\" target=\"_blank\" rel=\"noopener\">GitHub</a>."
+  hEasy: Fácil de Usar
+  pEasy: "<p class=\"grey px-md-5\">Hemos hecho toda esta sofisticación simple.<a href='https://docs.bisq.network/getting-started' target='_blank' rel=\"noopener\">Haga su primer intercambio</a> en menos de 10 minutos!</p>"
+  hBisqDao: La DAO Bisq
+  pBisqDao1: Gobernanza descentralizada para un exchange descentralizado.
+  pBisqDao2: Aprenda más aquí
+  hGetStarted: Cómo Empezar
+  divGetStarted: "Con Bisq, puede comenzar a intercambiar en menos de 10 minutos:"
+  liGetStarted1: Descargar la app
+  liGetStarted2: ", o construirla desde la fuente."
+  liGetStarted3: >
+    <li class="mt-0 mb-4 pb-4 border-bottom step step-text-2" data-image="step-image-2">Añada sus cuentas de moneda nacional o cripto</li> <li class="mt-0 mb-4 pb-4 border-bottom step step-text-3" data-image="step-image-3">Cree una nueva oferta o tome una existente.</li> <li class="mt-0 step step-text-4" data-image="step-image-4">Siga la interfaz de usuario para finalizar el intercambio.</li>
+  hJoin: Unirse a la comunidad
+  pJoin: >
+    <p>Bisq siempre está buscando desarrolladores altamente cualificados, diseñadores y expertos en seguridad para unirse a nuestra <a href="localhost:4001/community/">comunidad</a>. Si está interesado en ayudar a mejorar Bisq, hay muchas maneras de<a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuir</a>.
+     Nos encantaría<a href="localhost:4001/community/">saber de ti</a>!</p>
+    <p>Nuestra <a href="localhost:4001/roadmap/">hoja de ruta</a> proporciona una vista general de las características y hitos hasta llegar a la v.1.0. de Bisq</p>
```

Comment:
Looks like the period at the end of the sentence is misplaced:

`hasta llegar a la v.1.0 de Bisq.`